addon/styles/_variables.scss

```diff
@@ -0,0 +1,19 @@
+//
+// Variables used by
+//
+
+
+$frost-color-nav-bar-bg: $frost-color-night-2 !default;
+$frost-color-nav-border-left: rgba($frost-color-white, .2) !default;
+$frost-color-nav-category-border-right: rgba($frost-color-white, .2) !default;
+$frost-color-nav-category-active-bg: rgba(39, 36, 37, .5) !default;
+$frost-color-nav-column-border: rgba($frost-color-white, .3) !default;
+$frost-color-nav-modal-bg: rgba(51, 66, 79, .97) !default;
+$frost-color-nav-route-focus-bg: rgba(0, 0, 0, .2) !default;
+$frost-color-nav-route-active-bg: rgba($frost-color-nav-route-focus-bg, .05) !default;
+$frost-color-nav-route-focus-box-shadow: #fff !default;
+
+$frost-navigation-bar-height: 50px !default;
+$z-index-application-bar: 3 !default;
```

Comment:
FYI I've been trying to centralize all z-index values in core so that we know how they'll compete https://github.com/ciena-frost/ember-frost-core/blob/master/addon/styles/_z-index.scss